server/test/unit/test_plugin_manager.py

```diff
@@ -73,6 +73,7 @@ def test_types(self):
         """
 
         # Setup
+        types_db.clean()
```

Comment:
Yes, I should have mentioned that. One of the tests that I deleted cleans up after one of the tests that remains. When I removed the webpy tests, it caused this test to fail because it was operating on an unclean database.